util/db/migrations/020-create-returns.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Returns', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
```

Comment:
AutoIncrement is fine!
See Setts here:
https://github.com/Scottish-Natural-Heritage/standard-forestry-operations-api/blob/develop/util/db/migrations/20200326141910-sfo-create-sett.js#L7-L11

Reply:
Resolved in bfa0f693b2ca38a33ce26fd6a833da7c111a45e2.